examples/src/main/scala/org/apache/spark/examples/mllib/LDAExample.scala

```diff
@@ -174,7 +174,7 @@ object LDAExample {
 
     // Get dataset of document texts
     // One document per line in each text file.
-    val textRDD: RDD[String] = sc.textFile(paths.mkString(","))
+    val textRDD: RDD[String] = sc.textFile(paths.mkString(",")).coalesce(sc.defaultParallelism)
```

Comment:
I agree; this is a basic example.  I like the idea of adding a note telling users they could coalesce if they run this on small documents in practice, but I don't think we should complicate the code too much.

With one document per text file, the files could be pretty big; it just depends on document size.